contracts/CryptoChampions.sol

```diff
@@ -109,8 +113,8 @@ contract CryptoChampions is ICryptoChampions, AccessControl, ERC1155 {
         currentPhase = Phase.ONE;
     }
 
-    modifier isValidElderSpiritId(uint elderId) {
-        require(elderId > IN_GAME_CURRENCY_ID && elderId <= MAX_NUMBER_OF_ELDERS);  // dev: Given id is not valid.
```

Comment:
I've added a `modifier` for this in the branch I merged into main.  Should that be used here?

Reply:
Yea I didn't delete the modifier, only the parameter changed from uint to uint256.